frontend/src/metabase/routes.jsx

```diff
@@ -40,9 +40,8 @@ import UserCollectionList from "metabase/containers/UserCollectionList";
 
 import PulseEditApp from "metabase/pulse/containers/PulseEditApp";
 import SetupApp from "metabase/setup/containers/SetupApp";
-// new question
-import NewQueryOptions from "metabase/new_query/containers/NewQueryOptions";
-import NewDatasetOptions from "metabase/new_query/containers/NewDatasetOptions";
+
+import NewDatasetOptions from "metabase/new_model/containers/NewModelOptions";
```

Comment:
Can we use the `NewModelOptions` name here?

Reply:
Updated.